mifosng-android/src/instrumentTest/java/com/mifos/mifosxdroid/SplashScreenActivityTest.java

```diff
@@ -0,0 +1,27 @@
+package com.mifos.mifosxdroid;
+
+import androidx.test.internal.runner.junit4.AndroidJUnit4ClassRunner;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import static androidx.test.espresso.Espresso.onView;
+import static androidx.test.espresso.assertion.ViewAssertions.matches;
+import static androidx.test.espresso.matcher.ViewMatchers.isDisplayed;
+import static androidx.test.espresso.matcher.ViewMatchers.withId;
+
+import androidx.test.core.app.ActivityScenario;
+
+import com.mifos.utils.PrefManager;
+
+
+@RunWith(AndroidJUnit4ClassRunner.class)
+public class SplashScreenActivityTest {
+    @Test
+    public void testIntentToLoginActivity() {
+        ActivityScenario activityScenario = ActivityScenario.launch(SplashScreenActivity.class);

```

Comment:
It would be much better if you declare activity `ActivityScenario` variable globally and initialize it before the start of the test inside another method named `setup` which should be annotated by `Before` keyword.

Reply:
I will do that :)

Reply:
Done. Can you please take a look at it.